src/common/hooks/account/use-api-nonce.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { correctNextNonce } from '@common/hooks/account/use-api-nonce';
+import { AddressNonces } from '@stacks/blockchain-api-client/lib/generated';
```

Comment:
The thing is the api can  return this
```
{
detected_missing_nonces: [], 
last_executed_tx_nonce: null, 
last_mempool_tx_nonce: null, 
possible_next_nonce: 0, 
}
```
import { AddressNonces } from '@stacks/stacks-blockchain-api-types';

Does not allow null for the last_executed_tx_nonce attribute (and others)

import { AddressNonces } from '@stacks/blockchain-api-client/lib/generated';

does allow null.

This might be an issue with the blockchain-api-client.

Reply:
Ahh that sucks, yeah see that this is an issue with the types generation library. https://github.com/bcherny/json-schema-to-typescript/pull/312